source/support/check-organisation-email-address.html.erb

```diff
@@ -0,0 +1,56 @@
+---
+title: Check your organisation email address
+
+tags:
+
+description: Help and support connecting to GovWifi
```

Comment:
Should this be "Check your government or public sector email address to sign up to GovWifi." ?